.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8"]
+        python-version: ["3.8", "3.11"]
```

Comment:
@roquelopez Why do we need 3.8 here? Are there envs depends on py3.8?

Reply:
The CI will run the tests against all versions listed here. So, ideally, we should include all Python versions that we want to make sure that `alpha-automl` is compatible with. I think we should include all major versions from 3.8 through 3.11.